### DISCUSSION TITLE
How should postcss be configured outside of a plugin?

### DISCUSSION THREAD

**Discussion by IanVS** [facebook/docusaurus#7976] [Q&A] [answered]
Postcss is only mentioned in the docs in regards to plugin lifecycle methods.  I'd like to use [postcss-nesting](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-nesting) in my docusaurus project, but it seems that adding a `postcss.config.js` file at the root isn't sufficient to accomplish this.  What's the right way to configure postcss?  (I really hope it's not some kind of webpack config...)

**Comment by IanVS** [comment, marked as answer]
I take it back, adding the `postcss.config.js` file does work.  It would be cool if this was documented though.

**Reply by slorber** [reply]
hey @IanVS , we don't document this much on purpose, as the defaults should be good enough for most sites and we'd like to know what are your use-cases to customize it.

For example, anything std CSS should be added to our defaults.

Afaik CSS nesting std is looking for feedback on 3 possible final syntaxes. Once things are settled we can add this to our default postcss config. Until then, using plugin lifecycles or this root config file is the way.
